### DISCUSSION TITLE
j2commerce or j2cart?

### DISCUSSION THREAD

**Discussion by antoniotheant** [j2commerce/j2cart#103] [General]
I am a little confused, what is the difference between j2commerce and j2cart?

**Comment by niaziblog** [comment]
The development of J2Store has been stopped by the team
Now it has been forked by a stronger team and its new name is J2Commerce with more features and full support for Joomla 5 and soon Joomla 6.
I hope Joomla will become the best store builder(J2commerce) like WooCommerce
What Joomla can direct many users towards in this field

thanks from j2commerce team.

**Reply by antoniotheant** [reply]
Yes, I know. But what is j2cart?
![Screenshot 2024-11-18 alle 08 32 01](https://github.com/user-attachments/assets/f8e3728b-d040-4022-96f5-3a2722ae371c)

**Reply by niaziblog** [reply]
It is the same as J2Commerce, but it may become a part of J2Commercein the future
The good news is that J2Commerce is going to be much more advanced :)
